aggregator/src/aggregator/http_handlers.rs

```diff
@@ -168,7 +168,7 @@ impl Handler for Error {
 /// Firefox has the highest Max-Age cap, at 24 hours, so we use that. Our CORS preflight handlers
 /// are tightly scoped to relevant endpoints, and our CORS settings are unlikely to change.
 /// See: <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age>.
-const CORS_PREFLIGHT_CACHE_AGE: u32 = 24 * 60 * 60;
+const CORS_PREFLIGHT_CACHE_AGE: &'static str = "86400"; //24 * 60 * 60;
```

Comment:
This change is a infinitesimal small performance improvement, but also avoids calling `.to_string()`